code/src/Pages/Home.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState, useEffect} from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import LoadingSpinner from "Components/LoadingSpinner";
+import FilmList from "Components/FilmList";
+import Details from "Components/Details";
+import Error from "Components/Error";
+
+// Import URLS
+import { URLS } from "utils/urls";
+
+const Home = () => {
+
+    const [isLoading, setIsLoading] = useState(true)
+    const [popular, setPopular] = useState([])
+
+    useEffect(() => {
```

Comment:
We are really impressed about your code of how to fetch the API here using 'async', 'Promise.all()'. Well done!